File: Source/Back-end/src/main/java/org/elis/manutenzione/dto/response/GetManutenzioneDTO.java
package org.elis.manutenzione.dto.response;

import lombok.Data;

import java.time.LocalDate;


@Data
public class GetManutenzioneDTO {

    private long idManutenzione;
    private String nome;
    private String descrizione;
    private String utenteRichiedente;
    private String luogo;
    private String stato;
    private String manutentore;
    private LocalDate dataRichiesta;
    private LocalDate dataRiparazione;
    private LocalDate dataPrevista;
    private int priorita;
    private String immagine;
}

File: Source/Front-end/front-end/src/app/model/response/get-manutenzione-dto.ts
export class GetManutenzioneDTO {

  idManutenzione: number;
  nome: string;
  descrizione: string;
  utenteRichiedente: string;
  luogo: string;
  stato: string;
  manutentore: string;
  dataRichiesta: Date;
  dataRiparazione: Date;
  dataPrevista: Date;
  priorita: number;
  immagine: string;
}
